bin/tag

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+# shellcheck source=bin/_config.sh
+source "$(dirname "${BASH_SOURCE[0]}")/_config.sh"
+
+function usage(){
+    echo -e "Usage: ${0} [OPTIONS]
+
+Tag all untagged release commits (starting with the 🔖 gitmoji) and
+push the tags to github so the image is built and pushed to DockerHub.
+
+OPTIONS:
+  -h, --help       print this message
+  -c, --commit     dry run if this option is not set
+"
+}
+
+declare -i should_commit=0
+declare regex="🔖\(.*\).*version [0-9\.]*"
+
+# Parse options
+for i in "$@"
+do
+    case $i in
+        -h|--help|help)
+            usage "${0}"
+            exit 0
+            ;;
+        -c|--commit)
+            should_commit=1
+            shift
+            ;;
+        *)
+            shift
+            ;;
+    esac
+done
+
+
```

Comment:
You need  to `git fetch origin` first in case you didn't fetch tags created and pushed by one of our coworkers.

Reply:
nice catch :pray: